### PR TITLE
Doc: fixed a typo and added checked checkbox in Markdown Guide

### PR DESCRIPTION
**Pull request by mmahmoudian**

### CODE REVIEW

File: readme/markdown.md [hunk 1]
```diff
@@ -121,9 +121,15 @@ Note that Mermaid graphs are always rendered on a white background regardless of
...
-	- [ ] Rice
+	- [x] Rice
 	- [ ] Eggs
 
+Which would turn into:
+
+- [ ] Milk
+- [x] Rice
+- [ ] Eggs
+
```
Comment (tessus) [inline readme/markdown.md:132]: This probably won't work, since the `.md` files are translated into HTML for the web site.

Comment (mmahmoudian) [reply readme/markdown.md:132]: Well, it depends on what markdown flavor you use for static website generator. I assumed it is Github flavored. If not, I should either:
1. change it to a photo
2. remove this specific commit

Decision is yours as I don't know what do you use in the backend to generate static website.

Comment (tessus) [reply readme/markdown.md:132]: AFAIK Laurent wrote the generator or the tool that creates the html.

I've just tested your PR and that's how it turns out:

image

Either use an image and add it to `docs/images` or remove that section.

Comment (tessus) [reply readme/markdown.md:132]: and then reference it with `https://joplinapp.org/images/YOUR-IMG-NAME.jpg`

**Comment by mmahmoudian** [comment]
Considering that the image path was not relative or local, I could not confirm if the image loads correctly, but the syntax seems to be correct.

**Comment by tessus** [comment]
Thanks.

**Pull request merged**